lib/elixir/lib/task.ex

```diff
@@ -341,8 +341,17 @@ defmodule Task do
   Each `enumerable` item is passed as argument to the given function `fun` and
   processed by its own task. The tasks will be linked to the current process,
   similarly to `async/1`.
-
-  See `async_stream/5` for discussion, options, and examples.
+  
+  ## Example
+  
+  Count the graphemes in each string asynchronously, then add the counts together using reduce.
+  
+      iex> strings = ["long string", "longer string", "there are many of these"]
+      iex> stream = Task.async_stream(strings, fn text -> text |> String.graphemes |> Enum.count end)
```

Comment:
@josevalim that works as well, but I pointed at just `String.length/1` because it still counts graphemes, but it's just one function call and simplifies the code a bit, without the need for pipes and so on that can distract from the meat of the example.